changelog/changelog.ddoc

```diff
@@ -64,8 +64,10 @@ PHOBOSPR = $(PULL_REQUEST phobos,$1)
 _=
 
 CHANGELOG_VERSION = $(LI <a id="$1" href="$1.html">$1</a><span class="hide-from-nav"> ($2, $3)</span>)
+CHANGELOG_VERSION_PRE = $(LI <a id="$1" href="$1_pre.html">$1</a><span class="hide-from-nav"> (not yet released)</span>)
 _=BEGIN_GENERATED_CHANGELOG_VERSIONS
 CHANGELOG_VERSIONS =
+    $(CHANGELOG_VERSION_PRE 2.075.0, not yet released)
```

Comment:
"not yet released" is redundant here - you only use `$1`  in `CHANGELOG_VERSION_PRE`